lib/execution_engine2/db/models/models.py

```diff
@@ -336,7 +336,11 @@ class Job(Document):
         default=False
     )  # Marked true when all retry steps have completed
 
-    meta = {"collection": "ee2_jobs"}
+    # See https://docs.mongoengine.org/guide/defining-documents.html#indexes
+    # Hmm, are these indexes need to be + or - ?
+    indexes = [("status", "batch_job"), ("status", "-queued")]
```

Comment:
Hmm - would it make sense to have an index with the username, e.g. `(user, status, -queued)`? Does EE2 not do any user specific queries?

Reply:
Only for the job browser bff

```
        funcdef check_jobs_date_range_for_user(CheckJobsDateRangeParams params)
            returns (CheckJobsDateRangeResults) authentication required;
        funcdef check_jobs_date_range_for_all(CheckJobsDateRangeParams params)
            returns (CheckJobsDateRangeResults) authentication required;
```

Reply:
yeah I see that the original impetus was for the bad jobs reaper, which I assume doesn't care about the user name.

that being said, indexes targeted for those functions might not be a bad idea

Reply:
Here's another call that searches on clusterid instead of job id

https://github.com/kbase/execution_engine2/blob/1e268dfb1445e100da6fcb79219402953a32c0d1/lib/execution_engine2/sdk/EE2Status.py#L52-L53